src/testutil/build.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2018 The nvim-go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package testutil
+
+import (
+	"go/build"
+	"sync"
+	"testing"
+)
+
+var buildContextMu sync.Mutex
+
+func SetBuildContext(t *testing.T, gopath string) func() {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function SetBuildContext should have comment or be unexported